### DISCUSSION TITLE
Numbering for TOC and secondary sidebar

### DISCUSSION THREAD

**Discussion by DeeliN221** [jupyter-book/mystmd#1540] [Q&A]
![website-layout excal-e93b917409cfafd808d2559a4b4a3b5c](https://github.com/user-attachments/assets/06b8a844-7114-49a6-8842-f2ffb5a7464c)

Numbering headings works well, but how to attach numbers in the TOC and secondary sidebar?

**Comment by pancakereport** [comment]
You can add the following to your project frontmatter:
```
numbering:
  title: true 
  headings: true 
```

`title: true` yields numbering for primary sidebar. `headings: true` yields numbering for headings within a page. I'm not sure how to actually add numbering to the secondary sidebar. You can also set these within a page's frontmatter for more fine grained control.

Documentation: https://mystmd.org/guide/cross-references#title-numbering
